uint32 id
geometry_msgs/Point center
geometry_msgs/Quaternion quat
sensor_msgs/RegionOfInterest rect
float64 depth
uint8 left_brow_raise
uint8 right_brow_raise
uint8 left_brow_lower
uint8 right_brow_lower
uint8 smile
uint8 kiss
uint8 mouth_open
uint8 tongue_out
uint8 left_eye_closed
uint8 right_eye_closed
uint8 eyes_turn_left
uint8 eyes_turn_right
uint8 eyes_turn_up
uint8 eyes_turn_down
uint8 puff_left
uint8 puff_right
